src/client.js

```diff
@@ -0,0 +1,54 @@
+import isString from 'lodash/isString';
+import { Client as CoreClient, KinveyError, isDefined, Log } from 'kinvey-js-sdk/dist/export';
+let storage = Ti.App.Properties;
+
+class ActiveUserStorage {
+  constructor(client) {
+    if (isDefined(client.encryptionKey)) {
+      try {
+        // eslint-disable-next-line
+        const securely = require('bencoding.securely');
+        storage = securely.createProperties({
+          secret: client.encryptionKey
+        });
+      } catch (e) {
+        Log.debug('Unable to require the bencoding.securely module. Please install it to securely store the active user.', e);
```

Comment:
I don't think we want it to. If we allowed it to throw errors up then when someone updated to the latest release they would be required to install the `bencoding.securely` module. The other issue is that a developer has to provide the same encryption key each time to read the data stored by the module. Some developers might not care to do this. That is why I think we should try to use it if available and an `encryptionKey` is provided otherwise just fall back to `Ti.App.Properties`.